posts/2019/08/labs-dask-update.md

```diff
@@ -0,0 +1,21 @@
+<!--
+.. title: Labs Dask Update
+.. slug: labs-dask-update
+.. date: 2019-08-19
+.. author: James Bourbeau
+.. tags: Labs, Dask 
+.. category: 
+.. link: 
+.. description: 
+.. type: text
+-->
+
+This post provides an update on some recent [Dask](https://dask.org/)-related activities at Quansight.
+
+Through a community work order (CWO) for Dask development, the Quansight Labs team has been able to dedicate developer hours towards bug fixes and feature requests for Dask. These have touched on several portions of the Dask codebase, but generally have centered around bug fixes for Dask Arrays which enable larger-than-memory, distributed array computations. This work not only benefits the organization funding the development, but helps the entire project at large. To learn more about funding open source projects through CWOs, check out [this post](http://labs.quansight.org/blog/2019/05/community-driven-opensource-funded-development/) by Ralf Gommers which discusses the CWO model.
```

Comment:
Are you talking about the DE Shaw project here as a CWO? I think that's focused on specific features/bugs right? This either needs some more details, or needs to be removed (we're committed to some Dask core work also in the absence of external funding).

Reply:
Yes, that's the CWO I'm referring to. Would a list of some Dask PRs related to the CWO be an appropriate amount of detail? 

> we're committed to some Dask core work also in the absence of external funding

That's what the last paragraph is about

Reply:
> Would a list of some Dask PRs related to the CWO be an appropriate amount of detail?

that, or perhaps 1-2 sentences saying "key features are X and Y, we implemented those in PRs xx and yy"

Reply:
👍 I just pushed some commits which add more specifics about the CWO